lib/promisesdk.js

```diff
@@ -12,330 +12,342 @@ function ApigeeTool(defaults) {
   var cb = q.defer()
   var cmd = require('./commands/createappkey');
   runCommand(cmd, opts, cb);
   return cb.promise
 }
 
-ApigeeTool.attachFlowHook = function(opts) {
+ApigeeTool.attachFlowHook = function (opts) {
   var cb = q.defer()
   var cmd = require('./commands/attachFlowHook');
   runCommand(cmd, opts, cb);
   return cb.promise
 }
 
-ApigeeTool.detachFlowHook = function(opts) {
+ApigeeTool.detachFlowHook = function (opts) {
   var cb = q.defer()
   var cmd = require('./commands/detachFlowHook');
   runCommand(cmd, opts, cb);
   return cb.promise
 }
 
-ApigeeTool.getFlowHook = function(opts) {
+ApigeeTool.getFlowHook = function (opts) {
   var cb = q.defer()
   var cmd = require('./commands/getFlowHook');
   runCommand(cmd, opts, cb);
   return cb.promise
 }
 
-ApigeeTool.listRoles = function(opts) {
+ApigeeTool.listRoles = function (opts) {
   var cb = q.defer()
   var cmd = require('./commands/listRoles');
   runCommand(cmd, opts, cb);
   return cb.promise
 }
-ApigeeTool.createRole = function(opts) {
+ApigeeTool.createRole = function (opts) {
   var cb = q.defer()
   var cmd = require('./commands/createRole');
   runCommand(cmd, opts, cb);
   return cb.promise
 }
-ApigeeTool.getRole = function(opts) {
+ApigeeTool.getRole = function (opts) {
   var cb = q.defer()
   var cmd = require('./commands/getRole');
   runCommand(cmd, opts, cb);
   return cb.promise
 }
-ApigeeTool.deleteRole = function(opts) {
+ApigeeTool.deleteRole = function (opts) {
   var cb = q.defer()
   var cmd = require('./commands/deleteRole');
   runCommand(cmd, opts, cb);
   return cb.promise
 }
-ApigeeTool.setRolePermissions = function(opts) {
+ApigeeTool.setRolePermissions = function (opts) {
   var cb = q.defer()
   var cmd = require('./commands/setRolePermissions');
   runCommand(cmd, opts, cb);
   return cb.promise
 }
-ApigeeTool.getRolePermissions = function(opts) {
+ApigeeTool.getRolePermissions = function (opts) {
   var cb = q.defer()
   var cmd = require('./commands/getRolePermissions');
   runCommand(cmd, opts, cb);
   return cb.promise
 }
-ApigeeTool.assignUserRole = function(opts) {
+ApigeeTool.assignUserRole = function (opts) {
   var cb = q.defer()
   var cmd = require('./commands/assignUserRole');
   runCommand(cmd, opts, cb);
   return cb.promise
 }
-ApigeeTool.removeUserRole = function(opts) {
+ApigeeTool.removeUserRole = function (opts) {
   var cb = q.defer()
   var cmd = require('./commands/removeUserRole');
   runCommand(cmd, opts, cb);
   return cb.promise
 }
-ApigeeTool.verifyUserRole = function(opts) {
+ApigeeTool.verifyUserRole = function (opts) {
   var cb = q.defer()
   var cmd = require('./commands/verifyUserRole');
   runCommand(cmd, opts, cb);
   return cb.promise
 }
-ApigeeTool.listRoleUsers = function(opts) {
+ApigeeTool.listRoleUsers = function (opts) {
   var cb = q.defer()
   var cmd = require('./commands/listRoleUsers');
   runCommand(cmd, opts, cb);
   return cb.promise
 }
+ApigeeTool.updateTargetServer = function (opts) {
```

Comment:
As before, please move each function closer to its friends (where functions for other operations on same entity are)